<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Ecole</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="1" column="1">
       <widget class="QTableView" name="tableView">
        <property name="cursor" stdset="0">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>190</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGestion">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAjout_Dossier"/>
    <addaction name="actionDetails"/>
    <addaction name="actionSupprimer"/>
    <addaction name="actionEditer"/>
    <addaction name="actionGestion_Prestation"/>
   </widget>
   <addaction name="menuGestion"/>
   <addaction name="menuEdition"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAjout_Dossier"/>
   <addaction name="actionDetails"/>
   <addaction name="actionEditer"/>
   <addaction name="actionSupprimer"/>
   <addaction name="actionGestion_Prestation"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDossiers">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Academic-Hat.png</normaloff>:/images/Academic-Hat.png</iconset>
   </property>
   <property name="text">
    <string>Gestion Dossiers</string>
   </property>
  </action>
  <action name="actionPaiement_2">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Carton.png</normaloff>:/images/Carton.png</iconset>
   </property>
   <property name="text">
    <string>Gestion Prestations</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Info.png</normaloff>:/images/Info.png</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionBilan_Annuel">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Globe.png</normaloff>:/images/Globe.png</iconset>
   </property>
   <property name="text">
    <string>Bilan Annuel</string>
   </property>
  </action>
  <action name="actionBilan_Mensuel">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Globe.png</normaloff>:/images/Globe.png</iconset>
   </property>
   <property name="text">
    <string>Bilan Mensuel</string>
   </property>
  </action>
  <action name="actionDetails">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/ID.png</normaloff>:/images/ID.png</iconset>
   </property>
   <property name="text">
    <string>Details</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Cross.png</normaloff>:/images/Cross.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="actionEditer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Notepad.png</normaloff>:/images/Notepad.png</iconset>
   </property>
   <property name="text">
    <string>Editer</string>
   </property>
  </action>
  <action name="actionAjout_Dossier">
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Add.png</normaloff>:/images/Add.png</iconset>
   </property>
   <property name="text">
    <string>Ajout Dossier</string>
   </property>
  </action>
  <action name="actionRecherche">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Search.png</normaloff>
     <disabledoff>:/images/Search.png</disabledoff>:/images/Search.png</iconset>
   </property>
   <property name="text">
    <string>Recherche</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Cross.png</normaloff>:/images/Cross.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionGestion_Prestation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="logo.qrc">
     <normaloff>:/images/Carton.png</normaloff>:/images/Carton.png</iconset>
   </property>
   <property name="text">
    <string>Inscrire a une prestation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
